Sage/carbon
# feat(accordion): allow passing data-role prop to the root element - FE-2800

### Proposed behaviour
It should be possible to pass all props including `data-role` to the root element of the Accordion component

### Current behaviour
Currently it's not possible to pass any props that are not listed in prop-types.

### Checklist


- [ ] Screenshots are included in the PR

### Additional context
Closes #2933
### Testing instructions
Apply `data-role` prop using React DevTools and see if it's rendered on the Accordion
